DesignSafe-CI/portal
# task/DES-1964 - Publication Pipeline Wording Changes

## Overview: ##
A lot of minor wording and formatting changes for the amend/versioning pipeline

## PR Status: ##


## Related Jira tickets: ##

* [DES-1964](https://jira.tacc.utexas.edu/browse/DES-1964)

## Summary of Changes: ##

- On the last step of the "publish" process for other-type projects, there is no warning message that shows "You cannot make changes after publishing."

- Removed the word "title" from the last bullet of the "Version" section of the area in the publication pipeline where a user chooses if they want to publish, amend, or version.

- Replaced Error, Success, and Note messages w/ icons

- Change the "Prepare to Publish" buttons to display "Prepare to Amend or Version" for other-type projects that already have DOIs

- Added clarifying text to the end of the Versioning pipeline

-  Formatted the File Selection step within the Versioning Pipeline correctly.

## Testing Steps: ##
1. Create a new other-type project and take it through the publication pipeline, and make sure that the applicable changes above are visible
2. DM Frank to be added to an other-type project with a dummy DOI and take it through the Amends pipeline,  and make sure that the applicable changes above are visible
3. Using the same project with the dummy DOI, go through the Versioning pipeline and make sure that the applicable changes above are visible.

## UI Photos:

## Notes: ##
